cfg/config_default.cfg

```diff
@@ -1,29 +1,39 @@
 unbindall
 
-bind "ESCAPE" 		"cancelselect"
-bind "`" 		"toggleconsole"
-bind "w"		"+forward"
-bind "s"		"+back"
-bind "a"		"+moveleft"
-bind "d"		"+moveright"
-bind "SPACE"		"+jump"
-bind "CTRL"		"+duck"
-bind "e"		"+use"
-bind "q"		"+mouse_menu_taunt"
-bind "MOUSE1"		"+attack"
-bind "MOUSE2"		"+attack2"
-bind "F5"		"jpeg"
-bind "F6"		"save quick"
-bind "F7"		"load quick"
-bind "PAUSE" 		"pause"
-bind "TAB"		"+remote_view"
-bind "f" 		"+mouse_menu"
-bind "c" 		"+voicerecord"
-bind "t" 		"say"
-bind "MOUSE3" "+zoom"
-bind "MWHEELUP" "+zoom_in"
-bind "MWHEELDOWN" "+zoom_out"
-bind_osx "z" "+zoom"
-bind "r"		"swap_ss_input"
+// Movement
+bind "w"            "+forward"
+bind "s"            "+back"
+bind "a"            "+moveleft"
+bind "d"            "+moveright"
+bind "SPACE"        "+jump"
+bind "CTRL"         "+duck"
+bind "SHIFT"        "+speed"
+
+// Actions
+bind "MOUSE1"       "+attack"
+bind "MOUSE2"       "+attack2"
+bind "r"            "+reload"
+bind "g"            "+alt1" // Swap Portalgun/Paintgun
+bind "e"            "+use"
+bind "MOUSE3"       "+zoom"
+bind "MWHEELUP"     "+zoom_in"
+bind "MWHEELDOWN"   "+zoom_out"
+bind_osx "z"        "+zoom"
```

Comment:
(Just leaving a note) Platform specific binds are kinda nasty but I guess they make sense. It would be nice if controls were unified...

Reply:
Yeah, would be nice..